frontend/packages/ceph-storage-plugin/src/components/create-storage-system/payloads.ts

```diff
@@ -26,3 +26,30 @@ export const createSSPayload = (systemKind: string, systemName: string): Payload
     },
   };
 };
+
```

Comment:
```suggestion
spec: {
        name: systemName,
        kind: systemKind,
        namespace: CEPH_STORAGE_NAMESPACE,
      },
```
Line 23 payloads.ts. I guess you meant that

Reply:
yes! missed! Thanks

Reply:
We could use typed models for this later instead of generic payload to avoid these issues